src/components/Layout/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Link from "next/link";
+
+import styles from "./Navbar.module.scss";
+
+const PAGES = [
+  { path: "/", label: "Home" },
+  {
+    path: "/members",
+    label: "Our Members",
+  },
+  {
+    path: "/events",
+    label: "Events",
+  },
+  {
+    path: "/photos",
+    label: "Photos",
+  },
```

Comment:
How about disabling or hiding navbar links without corresponding pages? This is what I see when I click `Events` or `Photos`

<img width="869" alt="image" src="https://user-images.githubusercontent.com/24983797/204123148-d7ffc04b-a6e4-47c4-894d-275d28ec3098.png">

Reply:
I was thinking of adding some information in the events page, at least the weekly meetup info.
I'll do that today.
For the photos page we can probably do that one later, I'll hide it for now